release-notes/5.10.2.md

```diff
@@ -0,0 +1,51 @@
+# CiviCRM 5.10.2
+
+Released February 15, 2019
+
+- **[Synopsis](#synopsis)**
+- **[Bugs resolved](#bugs)**
+- **[Credits](#credits)**
+- **[Feedback](#feedback)**
+
+## <a name="synopsis"></a>Synopsis
+
+| *Does this version...?*                                         |         |
+|:--------------------------------------------------------------- |:-------:|
+| Fix security vulnerabilities?                                   |   no    |
+| Change the database schema?                                     |   no    |
+| Alter the API?                                                  |   no    |
+| Require attention to configuration options?                     |   no    |
+| Fix problems installing or upgrading to a previous version?     |   no    |
+| Introduce features?                                             |   no    |
+| **Fix bugs?**                                                   | **yes** |
+
+## <a name="bugs"></a>Bugs resolved
+
+- **[dev/core#681](https://lab.civicrm.org/dev/core/issues/681) - Fatal Error
+  Submitting Change Case Status form
+  ([13595](https://github.com/civicrm/civicrm-core/pull/13595))**
+
+  There was a fatal error generated when submitting the change case status form
+  because the end_date was not properly converted to MySQL date format.
+
+- **[dev/core#721](https://lab.civicrm.org/dev/core/issues/721) Fix Regression
+  on rebuilding smart group caches when ACLs are used.
+  ([13597](https://github.com/civicrm/civicrm-core/pull/13597))**
+
+  Due to some recent work on improving the loading of smart groups. Some code
+  changed such that the building of the smart group cache was happening after
```

Comment:
Fixed a recent regression whereby ability to see contacts permissioned by an ACL over a smart group depended on the freshness of the group cache